### PR TITLE
Add v1.6.0-cloud of mattermost-plugin-zoom to the Marketplace

### PR DESCRIPTION
**Pull request by agarciamontoro**
#### Summary
Generated with `go run ./cmd/generator/ add mattermost-plugin-zoom v1.6.0-cloud --official --cloud`.

Changelist at https://github.com/mattermost/mattermost-plugin-zoom/releases/tag/v1.6.0-cloud.

#### Ticket Link
--

### CODE REVIEW

**Review by larkox — APPROVED** [review]

**Review by saturninoabril — APPROVED** [review]
Yes @agarciamontoro and thanks! LGTM, will test after merged.

**Pull request merged**
